### ISSUE TITLE
Documents and contacts dont forward

### ISSUE THREAD

**Issue by apepenkov** [tgalal/yowsup#2853]
**Describe the bug**

While trying to forward message - document (```<class 'yowsup.layers.protocol_media.protocolentities.message_media_downloadable_document.DocumentDownloadableMediaMessageProtocolEntity'>```) or contact(```<class 'yowsup.layers.protocol_media.protocolentities.message_media_contact.ContactMediaMessageProtocolEntity'> ```): 
```Invalid token: 276``` (that's message id)
```
action = messageProtocolEntity.forward(msgRecipient)
                        self.toLower(action)
```
that works fine with images, text and videos

**Comment by apepenkov** [comment]
that error rises in encoder.py 
line 113
```
    def writeToken(self, token, data):
        if token <= 255 and token >=0:
            data.append(token)
        else:
            raise ValueError("Invalid token: %s" % token)
```

**Comment by apepenkov** [comment]
to fix documents:
```
    def writeToken(self, token, data):
        if token == 276:
            token = 103

        if token <= 255 and token >=0:
            data.append(token)
        else:
            raise ValueError("Invalid token: %s" % token)
```

**Comment by apepenkov** [comment]
Dont ask how i found 103 . It's retarded, but it works. BUT IT RQUIRES NORMAL FIX, THAT'S TEMP.

**Comment by apepenkov** [comment]
@tgalal ?

**Comment by bryanjhv** [comment]
It'd be way more useful if you submit a pull request for this if it's so important to you and think it'll be useful for everyone.

**Comment by apepenkov** [comment]
no. because that is shitty solution. I don't know wtf is token, and i randomly tried with different ones - 103 worked for docs, 101 worked for contacts. But token generator shall be fixed, that's temp fix by me
